zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -145,63 +143,41 @@ class ZiplineLoader(
     }
 
   /** Load application into Zipline without fallback on failure functionality. */
-  suspend fun load(
-    zipline: Zipline,
```

Comment:
Is it true that we generally don't want to be able to pass a Zipline instance in to be loaded and instead want to always use the Loader to both construct a Zipline and load the code?